docs/layout/dynamic-font-scaling.md

```diff
@@ -0,0 +1,205 @@
+# Dynamic Font Scaling
+
+Dynamic Font Scaling is a feature that allows users to choose the size of the text displayed on the screen. This helps users who need larger text for better readability, and it also accommodates users who can read smaller text.
+
+Dynamic Font Scaling is supported on Android, iOS, and iPadOS starting in Ionic v7.5.
+
+## Try It Out
+
+:::tip
+Be sure to try this on an Android, iOS, or iPadOS device.
+:::
+
+Follow the [Changing the Font Size on a Device](#changing-the-font-size-on-a-device) guide to set your preferred font size, and watch the text in the demo below grow or shrink according to your preferences.
+
+import DynamicFontScaling from '@site/static/usage/v7/layout/dynamic-font-scaling/index.md';
+
+<DynamicFontScaling />
```

Comment:
This doesn't work for me, unfortunately 🤔 I wonder if the iframe is causing problems, since the font size adjustment doesn't make it all the way through. I'm using Chrome on a Galaxy S10e.

|Before font scaling|After font scaling |
|--------------|---------------|
|![before](https://github.com/ionic-team/ionic-docs/assets/90629384/63acad1d-3d1f-4576-afbf-e45dfab20b28)|![after](https://github.com/ionic-team/ionic-docs/assets/90629384/5aaa363f-ad1c-4e48-8056-2237774282fa)|

Reply:
I updated the demo so it opens in a new tab/avoids iframes: https://github.com/ionic-team/ionic-docs/pull/3111/commits/d16510a95eb09db4c2ad4e0feb2c5fa25bfaf7e3

I also added a troubleshooting step because of this in case someone tries to load Ionic in an iframe: https://github.com/ionic-team/ionic-docs/pull/3111/commits/b37af9d598dbf1072e53aee75aa00662e4e34461

Reply:
We talked through this on Slack; turns out that there's a Chrome flag that's disabled by default on some devices (including mine) but is needed to make dynamic font scaling work. I confirmed that with the flag on, the playground now scales appropriately 👍 

Could we remove the troubleshooting item about not loading Ionic in an iframe, and replace it with one about ensuring the Chrome flag is enabled?

Reply:
Ok we demystified this a bit more, so I documented our findings here: https://github.com/ionic-team/ionic-docs/pull/3111/commits/85f54d04f17550bd917e6b63cdc6b823493f2f81

Can you give this a try again with the Accessibility Page Zoom feature enabled?